### PR TITLE
Add Line Gutter component

### PR DESCRIPTION
**Pull request by mattmolo**
Ability to stick custom items in the gutter of the logs

@nsat/sos

### CODE REVIEW

File: src/components/LineGutter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import React, { Component } from "react";
+
+import { lineGutter } from "./index.module.css";
+
+/**
+ * The line number of a single line.
+ * The anchor contained within is interactive, and will highlight the
+ * entire line upon selection.
+ */
```
Comment (Potaten2015) [inline src/components/LineGutter/index.tsx]: Is this for `LineNumber`

Comment (mattmolo) [reply src/components/LineGutter/index.tsx]: Well, I did copy paste this.. forgot to update this 🙃

**Review by Potaten2015 — APPROVED** [review]
Looks good minus rogue comment

**Comment by melloware** [comment]
Didn't know where else to post this but I have forked and published this component on NPM and it includes a lot of your updates here and MANY improvements.

GitHub: https://github.com/melloware/react-logviewer

Demo Site: https://melloware.github.io/react-logviewer/

**Pull request merged**